content/docs/concepts/network/index.md

```diff
@@ -0,0 +1,437 @@
+---
+title: The Istio network configuration objects and your application
+description: Describes Istio's network configuration objects and their functionality.
+weight: 30
+keywords: [network, virtual service, gateway, ingress, egress, service entry]
+aliases:
+---
+
+Cloud applications, unlike monolithic applications, are comprised of a
+collection of connected services. Throughout this document a service should be
+understood as an addressable destination workload inside the mesh. These
+services can connect to other services or to external resources. The Istio
+configuration objects allow you to configure traffic routes to and from
+services to secure the traffic between them. Other configuration objects
+provide telemetry and monitoring configuration options but this document
+focuses on the Istio's network configuration objects.
+
+The Istio network API defines four configuration objects: gateways, virtual
+services, destination rules, and service entries.
+The diagrams on this page don't show data plane traffic. Unless otherwise
+indicated, the arrows in the diagrams portray the traffic routes that the
+network configuration objects configured using destination rules. The diagrams
+show the possible control plane connections between the Envoy proxies of the
+network configuration objects.
+
+The following diagram shows the **configurable traffic routes** that the
+network configuration objects allow for a basic mesh with two different
+versions of a service:
+
+{{< image width="80%"
+    link="./net-config-1.svg"
+    caption="Istio network configuration objects overview"
+    >}}
+
+This document describes the key configurations each configuration object
+enables. Additionally, this document provides high-level examples of how the
+Istio network API can help you handle internal, incoming, and outgoing traffic.
+Lastly, the document discusses the benefits of using the Istio network API for
+your cloud applications.
+
+# The Istio network configuration objects
+
+The Istio network configuration objects provide you with four objects to
+configure the traffic routes between your services and your external resources:
+virtual services, destination rules, service entries, and gateways. You can add
+and configure these objects in a variety of ways to suit your needs. Each
+configuration object serves a specific purpose and provides specific
+configurations for your Istio service mesh. To create a solution for your
+specific mesh, you can combine the configuration objects in a way that best
+fulfills your needs.
+
+You can define the Istio network configuration objects as Kubernetes Custom
+Resource Definition [(CRD)](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/)
+objects and store their configuration in YAML files. The following sections
+describe the purposes and specific configurations of each configuration object
+and provide examples.
+
+## Virtual Services
```

Comment:
I followed a third approach, a didactic one. I began with the simplest and most familiar, virtual service, and went to the more complex and unfamiliar, service entries. There is a dependency between gateways and virtual services though. Because of it, I went with the virtual services first.